cmd/cli/exec/templates.go

```diff
@@ -0,0 +1,69 @@
+package exec
+
+import (
+	"fmt"
+	"strings"
+)
+
+var templateMap map[string]string = nil
+
+func init() {
+	templateMap = make(map[string]string)
+	templateMap["python"] = pythonTemplate()
+	templateMap["duckdb"] = duckdbTemplate()
+}
+
+func ErrUnknownTemplate(name string) error {
+	return fmt.Errorf("unknown template specified: %s", name)
+}
+
+// Template returns a string template (with yaml format) for the specified
+// job type. Initially these templates are encoded in this file, but in future
+// the first use of a name should load and cache a versioned template from the
+// server.
+func Template(name string) (string, error) {
+	tpl, found := templateMap[strings.ToLower(name)]
+	if !found {
+		return "", ErrUnknownTemplate(name)
+	}
+
+	return tpl, nil
+}
+
+func pythonTemplate() string {
+	t := `
+{"Name":"Python",
+ "Namespace":"default",
+ "Type":"batch",
+ "Count":1,
+ "Tasks":[{
+    "Name":"execute",
+	"Engine": {
+	    "Type":"python",
+		"Params":{
+			"Version": "{{or (index . "version") "3.11"}}"
+		}
+	}
+ }]
+}`
+
+	return strings.TrimSpace(t)
+}
```

Comment:
Could we do these as separate files (`templates/python.json`) and then `go:embed` them? Writing a map builder that just consumes the contents of an `fs.FS` (starting with an `embed.FS`) feels like a natural way to add support for templates in the repo later.

Reply:
Implemented in 799b33c9e
We can replace the loader with something that implements fs.ReadDirFS if we want to load these from a configured location in future (assuming also we have some way of installing them).

Reply:
Templated JSON turns out to not be valid JSON, and it isn't possibly to change the templates to use different '.  Renamed to tpl to avoid lint-issues but can rename back later